assignment06/script.js

```diff
@@ -0,0 +1,191 @@
+/*Heather Sleyster
+sleyster_assignment05b
+Thoedel
+INFO 2134
+4/9/2020
+*/
+
+//This is my event listener for the load event on the window.
+window.addEventListener('load', function() {
+/*These statements assign the element with the id values of userInput and
+convertFromHolder to the variables with the same name.    
+*/
+const userInput = document.getElementById('userInput');
+const convertFromHolder = document.getElementById('convertFromHolder');
+    /*This is the event listener for the change in the userInput field it has
+     a callback function onChange.
+    */
+userInput.addEventListener('change', onChange);
+    /*This is the onChange callback function. It checks the userInput and populates
+    the errors array if the change is empty or not a number with an error message.
+    It then itterates through the errors array and if an error is thrown creates a li
+    element in the errorHolder ul and adds the error message from the errors array to 
+    the li as a text node. If there are errors in the errors array the hidden class is 
+    removed from the errorHolder element and the class list of visible is added. If there
+    are no errors in the errors array the class of hidden is removed from the convertFromHolder
+    element and the class if visible is added.
+    */
+    function onChange(){
+        const errors = [];
```

Comment:
I would also set the error div back to not contain anything - like a reset. As you have it, once you have an error displayed - it doesn't go away - even when the input is valid.